judge/pdf_problems.py

```diff
@@ -13,12 +13,15 @@
 
 HAS_SELENIUM = False
```

Comment:
```suggestion
```

Reply:
Decided against because one may not have `USE_SELENIUM` set to `True`.